src/plugins/owner/SingleOwnerPlugin.sol

```diff
@@ -189,14 +196,13 @@ contract SingleOwnerPlugin is ISingleOwnerPlugin, BasePlugin, IERC1271 {
             dependencyIndex: 0 // Unused.
         });
         manifest.validationFunctions[6] = ManifestAssociatedFunction({
-            executionSelector: this.isValidSignature.selector,
-            associatedFunction: alwaysAllowRuntime
-        });
-        manifest.validationFunctions[7] = ManifestAssociatedFunction({
             executionSelector: this.owner.selector,
             associatedFunction: alwaysAllowRuntime
         });
 
+        manifest.signatureValidationFunctions = new uint8[](1);
+        manifest.signatureValidationFunctions[0] = uint8(FunctionId.SIG_VALIDATION);
```

Comment:
If we're assuming we don't need functionId, perhaps we can check if `manifest.validationFunctions.length` > 0, and if so, always allow `validateSignature` calls to that plugin.

Reply:
Yeah, really it should just be part of how validations are installed (a third option alongside user op and runtime validation), but that will have to be part of a later PR addressing user-supplied install configs: https://github.com/erc6900/resources/issues/9